tasks/extract_filepaths_for_granule/extract_filepaths_for_granule.py

```diff
@@ -41,9 +58,9 @@ def task(event, context):  # pylint: disable-msg=unused-argument
         Raises:
             ExtractFilePathsError: An error occurred parsing the input.
     """
-    LOGGER.debug(f"event: {event}")
+    LOGGER.error(f"event: {event}")
```

Comment:
now that your debugging is done change this to a `LOGGER.debug`. In normal production, we would only want to show the event information if we changed logging to DEBUG.

Reply:
reverted